packages/nimble/R/cppDefs_cppProject.R

```diff
@@ -268,17 +266,18 @@ cppProjectClass <- setRefClass('cppProjectClass',
                                        if(is.null(nimbleUserNamespace$sessionSpecificDll)) compileDynamicRegistrations(showCompilerOutput = showCompilerOutput)
 
                                        origSHLIBcmd <- SHLIBcmd
-                                       if(isTRUE(nimbleOptions()$stopCompilationBeforeLinking)) {## used only for testing, when we want to go quickly and skip linking and bail out
+                                       if(isTRUE(nimbleOptions('stopCompilationBeforeLinking'))) {## used only for testing, when we want to go quickly and skip linking and bail out
```

Comment:
Hmmm, not sure.  It looks like `getNimbleOption` will be more efficient and will error if `'myOption'` doesn't exist.  `nimbleOptions` can get or set.  It will be slower and it will return `NULL` if the option doesn't exist.  If we want to revamp this system let's do it via a different Issue & PR.  Am going to ignore for this unrelated PR.